### PR TITLE
Override Union meta

### PR DESCRIPTION
**Pull request by that-one-arab**
This gives us the option to add our own `_meta` to `Union`

Allowing us to implement features such as [this](https://github.com/graphql-python/graphene-django/pull/1537)

### CODE REVIEW

File: graphene/types/union.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def __init_subclass_with_meta__(cls, types=None, **options):
             isinstance(types, (list, tuple)) and len(types) > 0
         ), f"Must provide types for Union {cls.__name__}."
 
-        _meta = UnionOptions(cls)
+        _meta = options.pop("_meta", UnionOptions(cls))
```
Comment (erikwrede) [inline graphene/types/union.py:59]: Can you please add _meta as an explicit argument, similar to base.py or enum.py? I'd like to keep things consistent 🙂

**Review by erikwrede — CHANGES_REQUESTED** [review]
Thanks for the PR!

**Comment by that-one-arab** [comment]
@erikwrede Thank you for the quick review! I made the change please let me know if I did it correctly. Cheers!

**Review by erikwrede — APPROVED** [review]
Thank you! 😊

**Comment by erikwrede** [comment]
released in 3.4.2

**Pull request merged**